packages/form-js-editor/src/features/properties-panel/PropertiesPanelHeaderProvider.js

```diff
@@ -13,6 +13,9 @@ export function getPropertiesPanelHeaderProvider(options = {}) {
       if (headerlessTypes.includes(type)) {
         return '';
       }
+      if (type === 'datetime') {
+        return field.dateLabel || field.timeLabel;
```

Comment:
I wonder if we shouldn't merge both labels if they're both filled

Reply:
I think just the one is good enough for quick identification, which at the end of the day is all this little subheading is about (making sure you're changing the think you think you're working on). Maybe we can but it makes the headers kinda long so I'm happy leaving it like that personally.